src/Bridge/Symfony/Routing/IriConverter.php

```diff
@@ -92,7 +92,7 @@ public function getItemFromIri(string $iri, array $context = [])
     public function getIriFromItem($item, int $referenceType = UrlGeneratorInterface::ABS_PATH): string
     {
         $resourceClass = $this->getObjectClass($item);
-        $routeName = $this->routeNameResolver->getRouteName($resourceClass, false);
+        $routeName = $this->routeNameResolver->getRouteName($resourceClass, OperationType::ITEM, []);
```

Comment:
IMO we should because `Not defining it is deprecated since 2.1` and the `E_USER_DEPRECATED` are wrong if it's an optional argument.